Setup/UpgradeData.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Alma\MonthlyPayments\Setup;
+
+use Alma\MonthlyPayments\Gateway\Config\Config;
+use Alma\MonthlyPayments\Helpers\ApiConfigHelper;
+use Alma\MonthlyPayments\Helpers\Logger;
+use Alma\MonthlyPayments\Model\Adminhtml\Config\ApiUrl\CustomerCancelUrl;
+use Magento\Framework\App\ResourceConnection;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\UpgradeDataInterface;
+use Magento\Payment\Gateway\Config\Config as ConfigGateway;
+use \Exception;
+
+class UpgradeData implements UpgradeDataInterface
+{
+    /**
+     * @var Logger
+     */
+    private $logger;
+    /**
+     * @var CustomerCancelUrl
+     */
+    private $customerCancelUrl;
+    /**
+     * @var ResourceConnection
+     */
+    private $resourceConnection;
+
+    public function __construct(
+        Logger $logger,
+        CustomerCancelUrl $customerCancelUrl,
+        ResourceConnection $resourceConnection
+    )
+    {
+        $this->logger = $logger;
+        $this->customerCancelUrl = $customerCancelUrl;
+        $this->resourceConnection = $resourceConnection;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function upgrade(ModuleDataSetupInterface $setup, ModuleContextInterface $context)
+    {
+        $version = $context->getVersion();
+        if (version_compare($version, '2.3.2') < 0) {
```

Comment:
`version_compare($version, '2.3.2', "<")` seems more readable for me

Reply:
Done